library/core/src/num/uint_macros.rs

```diff
@@ -1136,6 +1174,38 @@ macro_rules! uint_impl {
             }
         }
 
+        /// Unchecked integer remainder. Computes `self % rhs`, assuming `rhs != 0`
+        ///
+        /// # Safety
+        ///
+        /// This results in undefined behavior when
+        #[doc = concat!("`rhs == 0`")]
+        /// i.e. when [`checked_rem`] would return `None`.
+        ///
+        #[doc = concat!("[`checked_rem`]: ", stringify!($SelfT), "::checked_rem")]
+        #[unstable(
+            feature = "unchecked_div_rem",
+            reason = "consistency with other unchecked_* functions",
+            issue = "136716",
+        )]
```

Comment:
```suggestion
        #[unstable(feature = "unchecked_div_rem", issue = "136716")]
```
`#[unstable(reason = "..")]` is typically not used, the tracking issue and ACP contain plenty of info